frontend/src/app/helpers/skeleton-loaders/mat-card-overview-loader/mat-card-overview-loader.component.spec.ts

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
Why was this file removed?

Reply:
I think I thought it was no longer needed, but I see it referenced in the project overview still.

Reply:
restored.